qemu/tests/live_snapshot_basic.py

```diff
@@ -37,7 +37,6 @@ def create_image(self):
         """
         image_name = self.params.get("image_name")
         self.params['image_name_snapshot'] = image_name + "-snap"
-        image_name_snapshot = self.params['image_name_snapshot']
```

Comment:
@PandaWei @xutian 

Yes, you are right. I have thought the previous line.

Acked-by: Cong Li <coli@redhat.com>

Thanks.